src/semidiscretization.jl

```diff
@@ -57,7 +60,10 @@ function Semidiscretization(mesh, equations, initial_condition, solver;
                             # while `uEltype` is used as element type of solutions etc.
                             RealT = real(solver), uEltype = RealT,
                             # tmp1 is needed for the `RelaxationCallback`
-                            initial_cache = (tmp1 = Array{RealT}(undef, nnodes(mesh)),))
+                            initial_cache = (tmp1 = Array{RealT}(undef, nnodes(mesh)),
+                                             tmp_partitioned = ArrayPartition(ntuple(_ -> zeros(real(solver),
+                                                                                                nnodes(mesh)),
+                                                                                     Val(nvariables(equations))))))
```

Comment:
```suggestion
                                             tmp_partitioned = allocate_coefficients(mesh, equations, solver))
```
Actually... why not just reuse `allocate_coefficients` for this?

Reply:
yes makes sense. This is also where I copied it from ...